tests/integration/linux_bridge_test.py

```diff
@@ -76,6 +77,14 @@ def bridge0_with_port0(port0_up):
         yield deepcopy(desired_state)
 
 
+@pytest.fixture
+def setup_vlan_iface(port0_up):
+    port_name = 'eth1.101'
```

Comment:
You should calculate it based on port0_up:
`iface_name = port0_up[Interface.KEY][0][Interface.NAME]] + '.' + vlan_id`